File: tests/fixtures/sample-docs.java
/**
 * Service for managing user accounts.
 */
public class UserService {
    /**
     * Create a new user with the given name and email.
     * @param name the user's display name
     * @param email the user's email address
     * @return the created User object
     */
    public User createUser(String name, String email) {
        return null;
    }

    /**
     * Delete a user by their ID. Returns true if successful.
     */
    public boolean deleteUser(int userId) {
        return false;
    }

    // Not a Javadoc comment
    public void logout() {}

    /** Maximum retry attempts */
    private int maxRetries = 3;
}

File: tests/fixtures/sample-docs.cs
/// <summary>
/// Service for managing user accounts.
/// </summary>
public class SampleDocs {
    /// <summary>
    /// Create a new user with the given name and email.
    /// </summary>
    public User CreateUser(string name, string email) {
        return null;
    }

    /// <summary>
    /// Delete a user by their ID.
    /// </summary>
    public bool DeleteUser(int userId) {
        return false;
    }

    // Not a doc comment
    public void Logout() {}

    /// <summary>Maximum retry attempts</summary>
    private int maxRetries = 3;
}
